.vale/styles/RedHat/TermsErrors.yml

```
@@ -20,6 +20,7 @@ swap:
   "(?<!Mozilla )Thunderbird": Mozilla Thunderbird
   "a lot(?: of)?": many|much
   "backward(?:-)?compatible": compatible with earlier versions
+  "[bB]asic [aA]uth": Basic HTTP authentication (first instance) then Basic authentication
```

like `lower left|lower-left` in the TermsErrors rule:

```suggestion
  "[bB]asic [aA]uth": Basic HTTP authentication (first instance)|Basic authentication
```

---

Thanks. Updated.